### PR TITLE
Wait for at most 100s on namespace deletion

### PR DESCRIPTION
**Pull request by IronPan**
Mitigate https://github.com/kubeflow/kubeflow/issues/2408

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2409)

### CODE REVIEW

File: scripts/kfctl.sh [hunk 1]
```diff
@@ -486,6 +486,9 @@ main() {
       kubectl delete ns/${K8S_NAMESPACE}
       while kubectl get ns/${K8S_NAMESPACE}; do
         echo "namespace ${K8S_NAMESPACE} not yet deleted. sleeping 10 seconds..."
+        # wait for 100s at most
+        # https://github.com/kubeflow/kubeflow/issues/2408
+        ((c++)) && ((c==10)) && break
```
Comment (kunmingg) [inline scripts/kfctl.sh]: We should give message ```delete failed``` instead of ```successfully deleted``` if we break here due to timeout

Comment (IronPan) [reply scripts/kfctl.sh]: done

File: scripts/kfctl.sh [hunk 2]
```diff
@@ -486,9 +486,16 @@ main() {
...
         sleep 10
       done
-      echo "namespace ${K8S_NAMESPACE} successfully deleted."
+      if kubectl get ns/${K8S_NAMESPACE}; then
+        echo "namespace ${K8S_NAMESPACE} successfully deleted."
```
Comment (kunmingg) [inline scripts/kfctl.sh]: Seems we should swap these 2 messages?

Comment (IronPan) [reply scripts/kfctl.sh]: done

**Comment by kunmingg** [comment]
/lgtm
/approve
thx!

**Pull request merged**